SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -142,14 +141,13 @@ class FDevAuthConfig : public FLocatorConfig
 
 	bool TryLoadCommandLineArgs()
 	{
-		bool bSuccess = true;
 		const TCHAR* CommandLine = FCommandLine::Get();
 		FParse::Value(CommandLine, TEXT("locatorHost"), LocatorHost);
 		FParse::Value(CommandLine, TEXT("deployment"), Deployment);
```

Comment:
To be honest deployment is a required field I believe so we should only succeed if a deployment and dev auth token are supplied. @joshuahuburn Agree with me please.

Reply:
The only usage I found is this: https://github.com/spatialos/UnrealGDK/blob/44cad2e1065ad1b753c05810e647c3c6af299b86/SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp#L208
And there is a check right after.

`DevelopmentAuthToken` is used here: https://github.com/spatialos/UnrealGDK/blob/44cad2e1065ad1b753c05810e647c3c6af299b86/SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp#L176 and only requires the token.

So I think we're fine?

Reply:
Ahh interesting. I think worker give back to use some random deployment to connect to lol. Ehh I can't decide if I like this or not.... @joshuahuburn choose for me.

Reply:
Yeah this isn't required per say, if you want to use dev-auth then it is required, normally we just get the deployment from the locator token.
Let's leave as is for now as this will error in the correct places if deployment is missing but dev-auth is used.